app/models/goal.py

```diff
@@ -2,4 +2,26 @@
 
 
 class Goal(db.Model):
-    goal_id = db.Column(db.Integer, primary_key=True)
+    goal_id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    title = db.Column(db.String, nullable=False)
+    tasks = db.relationship("Task", back_populates="goal", lazy=True)
+
+
+    def to_dict(self):
+        return{
+            "id":self.goal_id,
+            "title":self.title
+        }
+    
+    #create function for title  
+    @classmethod
+    def create(cls, request_body):
```

Comment:
Just for consistency's sake, it's a good idea to use a method name that is closer to from_dict to mirror our to_dict method!